src/components/ReplyField/ReplyFieldContainer.tsx

```diff
@@ -1,16 +1,28 @@
 import { connect } from 'react-redux';
-import { AppState, fetchCollaboratorsAction, getCollaborators, getCreatorCursor, setCursorAction } from '../../store';
+import {
+    AppState,
+    fetchCollaboratorsAction,
+    getCollaborators,
+    getCreatorCursor,
+    getFileId,
+    getIsCurrentFileVersion,
+    setCursorAction,
+} from '../../store';
 import ReplyField from './ReplyField';
 import { Collaborator } from '../../@types';
 
 export type Props = {
     collaborators: Collaborator[];
     cursorPosition: number;
+    fileId: string | null;
+    isCurrentFileVersion: boolean;
 };
 
 export const mapStateToProps = (state: AppState): Props => ({
     collaborators: getCollaborators(state),
     cursorPosition: getCreatorCursor(state),
+    fileId: getFileId(state),
```

Comment:
Should we use a `ReactRedux.useSelector` hook for this situation to avoid passing the prop down so far?